Code/EditorPlugins/Jolt/EditorPluginJolt/Dialogs/JoltProjectSettingsDlg.cpp

```diff
@@ -218,30 +218,29 @@ void ezQtJoltProjectSettingsDlg::on_DefaultButtons_clicked(QAbstractButton* pBut
 
 void ezQtJoltProjectSettingsDlg::on_ButtonAddLayer_clicked()
 {
-  const ezInt32 iNewIdx = m_Config.FindUnnamedGroup();
+  const ezUInt32 uiNewIdx = m_Config.FindUnnamedGroup();
```

Comment:
weird, I never noticed any issues here, how would the issue be reproduced ?

Reply:
Try adding a new collision layer. It will always say that a collision layer with that name already exists.